WordPress/WordPressTest/WordPressUnitTests.xctestplan

```diff
@@ -36,6 +36,13 @@
         "name" : "WordPressFluxTests"
       }
     },
+    {
+      "target" : {
+        "containerPath" : "container:..\/JetpackStatsWidgetsCore",
+        "identifier" : "JetpackStatsWidgetsCoreTests",
+        "name" : "JetpackStatsWidgetsCoreTests"
+      }
+    },
```

Comment:
For a target this small (one test class, two tests), it's not worth spinning up a dedicated VM in CI to run its tests.

We can get the same coverage by running the tests as part of the unit tests plan. 

_If_ we'll begin adding more and more modules, we might decide to have a "all unit tests" plan for CI and one "only app unit tests" for local development. That would come with the overhead that you'd have to _remember_ to switch scheme/test-plan depending on the module you are working in.